integration_tests/testsuite/test_monitoring.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+import json
+import logging
+import requests
+import time
+
+import test_util
+
+from google.cloud import monitoring as gcloud_monitoring
+
+
+def _test_monitoring(base_url):
```

Comment:
What we really want is this:

1) Text executable returns 0 or 1 to whatever ran it.

This is most useful thing.  Any automated test where you have to look at the logs to see if it passed, is never going to be looked at.  Whether you implement this via a testing framework, asserts, exceptions, or just sys.exit(1), I'm not too picky.

2) Finer granularity of tests and error output.  Once you have # 1 above, then it's nicer to have a finer granularity.  test_root worked but test_blah_blah didn't.  This is what a test framework gives you.

So, at least do # 1 by adding asserts or whatever.  If you want to leave # 2 for later, that's ok.  "unittest", "pytest", whatever you're comfortable with or want to learn.